src/gaussian_plot/mod.rs

```diff
@@ -0,0 +1,7 @@
+use gtk::glib;
+
+mod imp;
+
+glib::wrapper! {
```

Comment:
I wonder if it wouldn't better to not split those into two files :)

Reply:
I prefer it like this, seems cleaner to me :)